src/tests/functional/plugin/shared/src/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -102,6 +106,7 @@ TEST_P(OVPropertiesIncorrectTests, CanNotCompileModelWithIncorrectProperties) {
 }
 
 TEST_P(OVPropertiesDefaultTests, CanSetDefaultValueBackToPlugin) {
+    core->get_versions(target_device);
```

Comment:
`get_property` already initializes the plugin

Reply:
removed